optimum/utils/preprocessing/image_classification.py

```diff
@@ -0,0 +1,102 @@
+from functools import partial
+from typing import Dict, List
+
+import torch
+from datasets import Dataset, Metric, load_dataset
+from torchvision.transforms import CenterCrop, Compose, Normalize, Resize, ToTensor
+from transformers import FeatureExtractionMixin, ImageClassificationPipeline
+
+from .base import DatasetProcessing
+
+
+class ImageClassificationProcessing(DatasetProcessing):
+    def __init__(self, **kwargs):
+        if "secondary" in kwargs["data_keys"]:
+            raise ValueError("Only one data column is supported for token-classification.")
```

Comment:
It should be `image-classification` at the end of the string no?

Reply:
Yes, fixed, thanks!